op-node/eth/types.go

```diff
@@ -314,3 +314,12 @@ func (f FetchedReceipts) Result() (types.Receipts, error) {
 }
 
 var _ ReceiptsFetcher = (FetchedReceipts)(nil)
+
+// SystemConfig represents the rollup system configuration that carries over in every L2 block,
+// and may be changed through L1 system config events.
+// The initial SystemConfig at rollup genesis is embedded in the rollup configuration.
+type SystemConfig struct {
```

Comment:
Should be versioned?

Reply:
We don't need to version the struct as a whole, the individual config fields can have versions, and we can extend it with more fields later quite easily.